src/app/ngrx/reducers/podcast-downloads.reducer.ts

```diff
@@ -21,45 +20,32 @@ export function PodcastDownloadsReducer(state: PodcastDownloadsState = initialSt
   switch (action.type) {
     case ActionTypes.CASTLE_PODCAST_DOWNLOADS_LOAD: {
       const { id } = action.payload;
-      const currentIds = <string[]>selectPodcastDownloadsIds(state);
-      const charted = !includes(currentIds, id);
-      const podcastUpdate: UpdateStr<PodcastDownloads> = {
-        id,
-        changes: {
-          error: null,
-          loading: true,
-          loaded: false,
-          ...(charted && { charted }) // Conditionally add charted as object member
-        }
-      };
-      return adapter.upsertOne(podcastUpdate, state);
+      const entity = selectPodcastDownloadsEntities(state)[id];
```

Comment:
I temp broke this again in making these upgrade changes, so to review here is how `charted` should work: if the podcast entity has not been added to the state before, it should be charted. It is only false if the user has toggled it off, so the entity would already exist on state and be false.